build.gradle.kts

```diff
@@ -1,6 +1,11 @@
+import org.jlleitschuh.gradle.ktlint.tasks.KtLintCheckTask
+import org.jlleitschuh.gradle.ktlint.tasks.KtLintFormatTask
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
 plugins {
   id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.2.0"
-  kotlin("plugin.spring") version "1.8.21"
+  id("org.openapi.generator") version "6.6.0"
+  kotlin("plugin.spring") version "1.8.22"
   kotlin("plugin.jpa") version "1.8.21"
```

Comment:
Very minor, but if you are bumping `plugin.spring` could you also bump `plugin.jpa`?
I have no evidence to support this, but I believe these core kotlin plugins should be keep in sync in respect of their versions (though like I said I've not read that anyway, or have any other evidence - just seems "logical" to me?)

Reply:
Oops, I should have done that. Missed it in the merge conflict 👍